Source/WebGPU/WebGPU/WebGPU.h

```diff
@@ -1862,7 +1865,8 @@ WGPU_EXPORT void wgpuXRViewRelease(WGPUXRView view) WGPU_FUNCTION_ATTRIBUTE;
 WGPU_EXPORT WGPUBool wgpuXRViewIsValid(WGPUXRView view) WGPU_FUNCTION_ATTRIBUTE;
 
 #endif  // !defined(WGPU_SKIP_DECLARATIONS)
-
+using SpanConstUInt8 = std::span<const uint8_t>;
```

Comment:
This probably needs to be under a `ifdef _cplusplus` otherwise the SDK verifier will complain, unless the SDK verifier was complaining due to `[extern_c]` in the module map.